const.go

```diff
@@ -13,4 +13,8 @@ const (
 
 	// MaxSatoshi is the maximum transaction amount allowed in satoshi.
 	MaxSatoshi = 21e6 * SatoshiPerBitcoin
+
+	// Satoshi is the base bitcoin monetary unit.
+	// A single amount is equal to 1e-8 of a bitcoin.
```

Comment:
If there's confusion between NewAmount and type converting btcutil.Amount, let's rename NewAmount to something more specific (AmountFromBitcoin perhaps), not remove the ability to safely create satoshi integer amounts from floating points (which is necessary for the stupid wallet RPC api).